async-nats/src/jetstream/message.rs

```diff
@@ -88,6 +89,7 @@ impl Message {
             self.context
                 .client
                 .publish(reply.to_string(), "".into())
+                .into_future()
                 .map_err(Error::from)
                 .await
```

Comment:
These `.into_future()` calls shouldn't be needed anymore once `.map_err` and `.await` have swapped positions

Reply:
Good catch! Fixed!